deploy/docker/fs/opt/appsmith/run-java.sh

```
@@ -86,8 +86,10 @@ sh /opt/appsmith/run-starting-page-init.sh &
 
 # Ref -Dlog4j2.formatMsgNoLookups=true https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot
 exec java ${APPSMITH_JAVA_ARGS:-} ${APPSMITH_JAVA_HEAP_ARG:-} \
+  --add-opens java.base/java.lang=ALL-UNNAMED \
   --add-opens java.base/java.time=ALL-UNNAMED \
-  --add-opens java.base/java.nio=ALL-UNNAMED \
+  --add-opens java.base/java.nio=ALL-UNNAMED \  
```

Shellcheck (0.11.0)

[error] Delete trailing spaces after \ to break line (or use quotes for literal space).

(SC1101)